### PR TITLE
Polyhedron demo - add C3T3 `.tetra.am` reader

### PR DESCRIPTION
**Pull request by janetournois**
## Summary of Changes

Add a reader for tetrahedral meshes saved in `.tetra.am` files, ASCII or binary.
The mesh is loaded by `C3t3_io_plugin`.

The "save" function saves in ASCII mode.

## Release Management

* Affected package(s): Polyhedron_demo, Mesh_3/IO
* License and copyright ownership: unchanged

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp [hunk 1]
```diff
@@ -19,9 +19,11 @@ class Polyhedron_demo_c3t3_binary_io_plugin :
 
 public:
   QString name() const override { return "C3t3_io_plugin"; }
-  QString nameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma)"; }
-  QString saveNameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma);;avizo (*.am);;OFF files (*.off)"; }
-  QString loadNameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh)"; }
+  QString nameFilters() const override { return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma);;avizo (*.tetra.am)"; }
+  QString saveNameFilters() const override{
+    return "binary files (*.cgal);;ascii (*.mesh);;maya (*.ma);;avizo (*.tetra.am);;OFF files (*.off)"; }
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp:24]: Can we really save to `.tetra.am` files?

Comment (janetournois) [reply Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp:24]: yes we can!

Comment (janetournois) [reply Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp:24]: using `CGAL::IO::output_to_avizo()`

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.6-Ic-88

**Pull request merged**